packages/config/config/devices/0x019b/zm_single_relay_16.json

```
@@ -1,8 +1,8 @@
 {
-	"manufacturer": "HeatIt",
+	"manufacturer": "Heatit",
 	"manufacturerId": "0x019b",
-	"label": "4512671",
-	"description": "Heatit ZM Single Relay 16A",
+	"label": "ZM Single Relay 16A",
+	"description": "16A Single Z-wave relay",
```

```suggestion
	"label": "ZM Single Relay 16A",
	"description": "High power relay",
```
They put this on the box, which is better than repeating the label IMO.